masterlist.yaml

```
@@ -514,6 +514,11 @@ common:
     subs:
       - 'CC Survival Mode'
       - '[Survival Mode Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/19154/)'
+  - &patch3rdParty_ClockA
+    <<: *patch3rdParty
+    subs:
+      - "[Clockwork](https://www.nexusmods.com/skyrimspecialedition/mods/4155/)"
+      - "[Additional Clockwork](https://www.nexusmods.com/skyrimspecialedition/mods/47087)"
```

Some formatting changes for consistency with the rest of the masterlist:

```suggestion
      - '[Clockwork](https://www.nexusmods.com/skyrimspecialedition/mods/4155/)'
      - '[Additional Clockwork](https://www.nexusmods.com/skyrimspecialedition/mods/47087/)'
```